base/base/sort.h

```diff
@@ -2,6 +2,64 @@
 
 #include <pdqsort.h>
 
+#ifdef NDEBUG
```

Comment:
NDEBUG means No Debug and is set in Release mode, you need to use ifndef everywhere

Reply:
Need to enable this by default, to check if we have issues with tests that run only in release.